src_assets/common/assets/web/configs/tabs/audiovideo/DisplayDeviceOptions.vue

```diff
@@ -108,7 +108,7 @@ const config = ref(props.config)
                 <label for="dd_config_revert_delay" class="form-label">
                   {{ $t('config.dd_config_revert_delay') }}
                 </label>
-                <input type="number" class="form-control" id="dd_config_revert_delay" placeholder="3000" min="0"
+                <input type="text" class="form-control" id="dd_config_revert_delay" placeholder="3000"
```

Comment:
```suggestion
                <input type="number" class="form-control" id="dd_config_revert_delay" placeholder="3000" min="0"
```

Please keep this change that came with the reverted PR